### ISSUE TITLE
Manually disable automatic scroll bar

### ISSUE THREAD

**Issue by natcohen** [rintoj/ngx-virtual-scroller#278]
We should be able to manually disable the scroll bar added in `public set parentScroll(value: Element | Window)`

        if (!disableAutomaticScroll && scrollElement !== this.element.nativeElement) {
            this.oldParentScrollOverflow = { x: scrollElement.style['overflow-x'], y: scrollElement.style['overflow-y'] };
            scrollElement.style['overflow-y'] = this.horizontal ? 'visible' : 'auto';
            scrollElement.style['overflow-x'] = this.horizontal ? 'auto' : 'visible';
        }

I already manage myself the scroll bar and don't want this module to deal with it...

**Comment by speige** [comment]
I agree with this. Would you mind submitting a pull request to add this if statement as well as the new disableAutomaticScroll property? Also, there should be a mention of the new property in the readme.

Thanks!

I wonder if there's a better name for the property. It might be hard to know what it means without reading the documentation. Maybe something like "modifyOverflowStyleOfParentScroll" which defaults to true?

So, the if statement would be the opposite:
```
    if (modifyOverflowStyleOfParentScroll && scrollElement !== this.element.nativeElement) {
```

I don't know a good variable name, it's just that disableAutomaticScroll makes me imagine that the scrollbar is moving by itself & this property disables that, which isn't correct.

**Comment by natcohen** [comment]
@speige I have added PRs. Shouldn't we change the version and add it to the changelog?

**Comment by speige** [comment]
Thanks. They've been merged. I'll update the version when I release to npm.

**Comment by speige** [comment]
All of your changes have been published to npm under v1.0.14

Thanks for your help!
